crates/c-api/include/wasi.h

```diff
@@ -163,6 +163,28 @@ WASI_API_EXTERN bool wasi_config_set_stderr_file(wasi_config_t *config,
  */
 WASI_API_EXTERN void wasi_config_inherit_stderr(wasi_config_t *config);
 
+/**
+ * \brief This directory can be read, for example its entries can be iterated
+ * over and files can be opened.
+ */
+#define WASMTIME_WASI_DIR_PERMS_READ 1
```

Comment:
I didn't see any other `#defines` for enums that I could use for a naming convention, so let me know if these should be moved or renamed.

Reply:
Looks good to me!